src/prompt.rs

```diff
@@ -1,11 +1,13 @@
 /// Set default pixi prompt for the bash shell
 pub fn get_bash_prompt(env_name: &str) -> String {
```

Comment:
Maybe we should rename this to `get_bash_init_script` or something. Its no longer just setting the PS1.